Added contribution section to the docs

JulStraus opened the pull request:
I added a contribution section to the documentation. The section is quite a lot inspired by the one of [`EnergyModelsBase`](https://energymodelsx.github.io/EnergyModelsBase.jl/stable/how-to/contribute/), but I think i removed all references to EMX packages after the copy-paste.

In its core, it is inspired by the one of JuMP.

hellemo commented on docs/src/contribute.md:
```diff
@@ -0,0 +1,85 @@
+Feature requests can be achieved through two approaches:
+
+1. create an issue describing the aim of the feature request and
+2. incorporate the feature request through a fork of the repository.
```
```suggestion
2. incorporate the feature request through a fork of the repository and open a pull request.
```

trulsf commented on docs/src/contribute.md:
```diff
@@ -0,0 +1,85 @@
+We will internally assign the relevant person to the pull request.
+
+You may receive quite a few comments with respect to the incorporation and how it may potentially affect other parts of the code.
+Please remaing patient as it may take potentially some time before we can respond to the changes, although we try to answer as fast as possible.
```
```suggestion
Please remain patient as it may take potentially some time before we can respond to the changes, although we try to answer as fast as possible.
```

trulsf commented on docs/src/contribute.md:
```diff
@@ -0,0 +1,85 @@
+
+## [File a bug report](@id con-bug_rep)
+
+Another approach to contributing to `TimeStruct` is through filing a bug report as an *[issue](https://github.com/sintefore/TimeStruct.jl/issues/new)* when unexpected behaviour is occuring.
```
```suggestion
One approach to contributing to `TimeStruct` is through filing a bug report as an *[issue](https://github.com/sintefore/TimeStruct.jl/issues/new)* when unexpected behaviour is occuring.
```

JulStraus commented:
I added all changes. From my end, it is good to merge. I leave it to you @trulsf  to merge it once @hellemo has approved as well (if you consider it required).

A side note:
We moved to EMX to test x64 LTS and latest now on both Windows and Linux while we removed nightly and x86 (it does not really makes sense as the first x86-64 CPUs came out in 2003). We could consider including ARM tests, if supported. This requires adjustments to the Julia Action versions as can be seen in [this PR of EMRP](https://github.com/EnergyModelsX/EnergyModelsRenewableProducers.jl/pull/27/files).
